src/components/news/CreateNews.js

```
@@ -0,0 +1,773 @@
+          {selectedTags.map((tag, index) => (
+            <span key={index} className="preview-tag">{tag}</span>
+          ))}
+        </div>
+
+        {imagePreview && (
+          <div className="preview-image">
+            <img src={imagePreview} alt={t('news.create.previewAlt') || 'News preview'} />
+          </div>
+        )}
+
+        <div className="preview-text" dangerouslySetInnerHTML={{ __html: text }}></div>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)